.github/workflows/.ecs-deploy-task-definition.yml

```diff
@@ -110,7 +114,7 @@ jobs:
           set -e  
 
           export env="${{ inputs.deploy_env }}"
-          export source_path="main/${{ inputs.source_path }}deployment/definition.yml"
+          export source_path="main/${{ inputs.source_path }}/${{ inputs.task_definition_path }}"
```

Comment:
Why an addition / here?

Reply:
Mistake, thanks for pointing out.
Previously I change the `/definition.yml` part je, didn't remove it just now